src/nz-zorro.scss

```
@@ -28746,7 +28746,7 @@ nz-tree {
     text-align: center;
 }
 .ant-result-subtitle {
-    color: rgba(0, 0, 0, 0.45);
+    color: #777;
```

```suggestion
    color: $devui-text;
```
As we're using devui variables as a base (to be changed), it would be much better for us to change colors according to mode, instead of fixing to one.

Refer: `ng-devui/styles-var/devui-var.scss`